packages/interaction-tracking/src/InteractionTracking.js

```diff
@@ -195,7 +195,7 @@ export function wrap(
     interaction.__count++;
   });
 
-  const wrapped = () => {
+  const wrapped = (...args) => {
```

Comment:
You could also change this to a normal function :D

Reply:
It's a nit. I'm just paranoid about the Babel output.

Reply:
The Babel output is fine. I've checked it. 😄

But sure, I don't mind.